PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+# Goal
+
+[JIRA Ticket](https://jira)
+
+## What's Here
+
+> Describe what you did and why, before and after.
+
+## Steps to Test
+
+> How can a reviewer test and run the code through its paces?
```

Comment:
I would add this as an optional section in case there are enough unit/functional/other tests.